src/posts/2020-10-28-dyslexia-fonts.md

```diff
@@ -0,0 +1,64 @@
+---
+layout: post
+title: Dyslexia fonts
+description: A quick overview of dyslexia fonts
+category: Assistive technology
+author: Hamsa Harcourt
+further_reading:
+  - url: https://accessibility.huit.harvard.edu/design-readability
+    title: Design for readability
+    source: Havard University
+date: 2020-10-28
+eleventyNavigation:
+  key: {{ category }}
+  title: {{ title }}
+tags:
+  - assistivetechnology
+---
+
+According to Oxford Advanced Learner's Dictionary:
+
+<blockquote>
+  <p>
+    Dyslexia is a slight disorder of the brain that causes difficulty in reading and spelling, for example, but does not affect intelligence.
+  </p>
+</blockquote>
+
+Dyslexia affects about 1 in 5 people worldwide. Its symptoms range in severity from mild to extreme. Since letters, numbers, punctuation, and other characters  are often transposed by Dyslexia, font choice is an important thing to consider.
+
+
+## Background
+
+When designing fonts, some typographers flip and rotate one character to form other characters. Letters b, p, d and q are usually created using this method. This is because it makes it easier to make the font. However, this practice can causes problems for people with Dyslexia. This is because some letters, numbers, punctuation, and other characters become indistinguishable. Hence, making it easy to confuse one for another. As a result, people with Dyslexia prefer fonts where all the characters are unique. 
+
+
+## Introducing Dyslexia fonts
+
+Dyslexia fonts are typefaces designed specially for people with Dyslexia. They are designed so that each character is unique and distinguishable. Dyslexia fonts also make each character unique by using thicker lines in some parts of the letters, as well as slanting them a bit. 
+
+Some Dyslexia fonts include: 
+
+- [Dyslexie](https://www.dyslexiefont.com/en/home/), 
+- [Open Dyslexic](https://opendyslexic.org/), 
+- [Lexie Readable](http://www.k-type.com/fonts/lexie-readable/), and
+- [Read Regular](http://www.readregular.com/english/intro.html).
+
+![an example of Dyslexia fonts](../img/posts/2020-10-28-dyslexia-fonts/dyslexia-font.png)
+ 
+According to recent studies, it has been proven that Dyslexia fonts do **not** help people **read faster and make fewer mistakes**. In fact, most people with Dyslexia prefer Verdana and Helvetica to  other fonts. For  a  detailed  review of the case study, see  [Good Fonts for Dyslexia (opens PDF)](http://dyslexiahelp.umich.edu/sites/default/files/good_fonts_for_dyslexia_study.pdf)
```

Comment:
What about Comic Sans? I know some people prefer it, as well. Here's a good post on the subject I was just reading: https://www.thecut.com/2020/08/the-reason-comic-sans-is-a-public-good.html

Reply:
Thank you for your observation. The resource was also helpful. Additionally, I've done a quick research about comic sans. I think I'll add it to the list.